.gitignore

```diff
@@ -1,5 +1,7 @@
 src/database/migrations/
 src/database/seeders/
 node_modules/
+
+packageScript.json
```

Comment:
What is this intended for?

Reply:
This is for personal use. I have stored my script in `packageScript.json`.

Reply:
if this script is used for migrations, should be in package.json on the scripts part. If it's unrelated to the app it shouldn't be on folder